src/hooks/useConnectVenueChats.ts

```diff
@@ -0,0 +1,14 @@
+import { useFirestoreConnect } from "react-redux-firebase";
+
+export const useConnectVenueChats = (venueId?: string) => {
+  useFirestoreConnect(
+    venueId
```

Comment:
As a suggestion, maybe we can try with guards instead of conditionals. That way useFirestoreConnect won't be called at all. I suppose no call is better than a function call with undefined.

```jsx
export const useConnectVenueChats = (venueId?: string) => {
  if (!venueId) {
    return;
  }

  useFirestoreConnct(query)
}
```

Reply:
Good suggestion. This has the problem that you can't conditionally call a hook, unfortunately. So we have to put it all inside the `useFirestoreConnect` call.